test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -414,14 +434,18 @@ class test_uct_cm_sockaddr : public uct_test {
                        user_data);
     }
 
-    static void
-    cm_conn_request_cb(uct_listener_h listener, void *arg,
-                       const char *local_dev_name,
-                       uct_conn_request_h conn_request,
-                       const uct_cm_remote_data_t *remote_data) {
+
+    static int cm_common_conn_request_cb(uct_listener_h listener, void *arg,
```

Comment:
return type `bool` and pls document how to handle it + rename, actually, this is not a callback